tcl/pd_i18n.tcl

```diff
@@ -56,7 +56,11 @@ proc ::pd_i18n::get_system_language {} {
     } else {
         set lang ""
     }
-    return [file rootname [string tolower $lang]]
+    set lang [file rootname [string tolower $lang]]
+    if { $lang == "c" } {
```

Comment:
good catch, I can see "c" causing problems...

Reply:
?
i committed this mostly for cosmetic reasons...

Reply:
I was imagining a platform that might not match the locale due to case-sensivity (or lack thereof). ;)